docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,335 @@
+---
+description: Learn how Prefect deployments can be configured for scheduled and remote execution with work pools.
+tags:
+    - work pools
+    - orchestration
+    - flow runs
+    - deployments
+    - schedules
+    - tutorial
+search:
+  boost: 2
+---
+
+# Work Pools
+
+!!! note "Prefect Cloud"
```

Comment:
@discdiver what you you think of moving this warning to the "Set up a work pool" section and just having the user skip to the next tutorial from there if appropriate? I think it would read more clearly.

Reply:
That's a good call. Thank you. Done.